chainerx_cc/chainerx/cuda/cuda_device/std_ops.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
```

Comment:
Why not include the definitions in these files to the existing `{native,cuda}/op_regist.h` ?

Reply:
Oh my bad, since the files that I changed had them, I didn't notice the missing header. Thanks.

Reply:
Sorry for pinging but I understand this is still a TODO?

Reply:
Its been taken care off

Reply:
Ah, what I meant with this comment in the first place was that the `std_ops.h` are not needed and that the content of this file can be moved to the existing `op_regist.h` instead.

Reply:
Oh sorry, I misunderstood that. Will fix it in few hours